kinesumer.go

```diff
@@ -222,6 +228,10 @@ func (kin *Kinesumer) Begin() (int, error) {
 
```

Comment:
Hmm, have this sleep a random amount of time between 500ms to 2000ms, So that processes that start at the same time won't be synchronized.